src/SubMenu.js

```diff
@@ -1,42 +1,58 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
+import AbstractMenu from './AbstractMenu';
 import { cssClasses, hasOwnProp } from './helpers';
 
-export default class SubMenu extends Component {
+export default class SubMenu extends AbstractMenu {
     static propTypes = {
         children: PropTypes.node.isRequired,
         title: PropTypes.node.isRequired,
         className: PropTypes.string,
         disabled: PropTypes.bool,
         hoverDelay: PropTypes.number,
-        rtl: PropTypes.bool
+        rtl: PropTypes.bool,
+        selected: PropTypes.bool,
+        onMouseMove: PropTypes.func.isRequired,
+        onMouseOut: PropTypes.func.isRequired,
+        forceOpen: PropTypes.bool.isRequired,
+        forceClose: PropTypes.func.isRequired,
+        parentKeyNavigationHandler: PropTypes.func.isRequired
```

Comment:
Would be nice to know in which code context the required option is a problem for you. Would you prefer a code comment documentation or an external documentation file?